ajax/libs/Web-Ticker/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Web-Ticker",
+  "version": "3.0.0",
+  "description": "JQuery Web Ticker",
+  "filename": "jquery.webticker.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mazedigital/Web-Ticker.git"
+  },
+  "author": {
+    "name": "Jonathan Mifsud",
+    "email": "jon@maze.digital",
+    "url": "https://maze.digital"
+  },
+  "homepage": "https://maze.digital/webticker/",
+  "keywords": [
+    "animation",
+    "ecosystem",
+    "ecosystem-jquery",
+    "horizontal",
+    "horizontal-scroll",
+    "image",
+    "image-scroll",
+    "jquery",
+    "jquery-plugin",
+    "marquee",
+    "news",
+    "news-ticker",
+    "plugin",
+    "scroll",
+    "ticker"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mazedigital/Web-Ticker.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.webticker*"
+        ]
+      }
+    ]
+  },
+  "licenses": [
```

Comment:
Lets leave only GPL-3.0

Reply:
@sashberd cdnjs need commercial license if there is one. See https://github.com/cdnjs/cdnjs/pull/11551#issuecomment-322674874